### ISSUE TITLE
new element android 1.5.6 can not login on matrix-synapse 1.50.2+buster1

### ISSUE THREAD

**Issue by progserega** [vector-im/element-android#7529]
### Steps to reproduce

1. update android element to latest v1.5.6
2. start app
3. see whole green screen
4. wait ~5 minutes - no changes
5. clear app data
6. reboot phone, start element
7. enter my server and login/password
8. see "first sync..." splash
9. wait 10 minutes - no any changes
10. in servers logs I see:
```
    2022-11-06 13:23:03,962 - synapse.access.http.8008 - 445 - INFO - GET-4214899 - ::1 - 8008 - {None} Processed request: 0.003sec/0.000sec (0.001sec, 0.000sec) (0.000sec/0.001sec/2) 84B 200 "GET /_matrix/client/r0/profile/@progserega:rsp    __rim.ru HTTP/1.0" "Element/1.5.6 (OnePlus KB2003; Android 12; KB2003_11_C.36; Flavour GooglePlay; MatrixAndroidSdk2 1.5.6)" [0 dbevts]
    2022-11-06 13:23:04,025 - synapse.http.server - 95 - INFO - GET-4214900 - <SynapseRequest at 0x7fe1ec07e128 method='GET' uri='/_matrix/client/r0/sync?filter=%7B%22room%22%3A%7B%22state%22%3A%7B%22lazy_load_members%22%3Atrue%7D%2C%22tim    __eline%22%3A%7B%22unread_thread_notifications%22%3Atrue%7D%7D%7D&set_presence=offline&timeout=0' clientproto='HTTP/1.0' site='8008'> SynapseError: 400 - Additional properties are not allowed ('unread_thread_notifications' was unexpect    __ed)
    
    Failed validating 'additionalProperties' in schema['properties']['room']['properties']['timeline']:
        {'additionalProperties': False, 
         'properties': {'contains_url': {'type': 'boolean'},
                        'include_redundant_members': {'type': 'boolean'},
                        'io.element.relation_senders': {'items': {'type': 'string'},
                                                        'type': 'array'},
                        'io.element.relation_types': {'items': {'type': 'string'},
                                                      'type': 'array'},
                        'lazy_load_members': {'type': 'boolean'},
                        'limit': {'type': 'number'},
                        'not_rooms': {'$ref': '#/definitions/room_id_array'},
                        'not_senders': {'$ref': '#/definitions/user_id_array'},
                        'not_types': {'items': {'type': 'string'},
                                      'type': 'array'},
                        'org.matrix.labels': {'items': {'type': 'string'},
                                              'type': 'array'},
                        'org.matrix.not_labels': {'items': {'type': 'string'},
                                                  'type': 'array'},
                        'rooms': {'$ref': '#/definitions/room_id_array'},
                        'senders': {'$ref': '#/definitions/user_id_array'},
                        'types': {'items': {'type': 'string'},
                                  'type': 'array'}},
         'type': 'object'}
    
    On instance['room']['timeline']:
        {'unread_thread_notifications': True}
```

#### What did you expect?
New version of element must work with any version of server. 
#### What happened instead?
Any update can broke corp installation.

### Your phone model

OnePluse 8t

### Operating system version

Android 12

### Application version and app store

element andrid 1.5.6

### Homeserver

rsprim.ru

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by progserega** [comment]
Why element-android not check API-version for select supported scheme in server?
Otherwise updating - is lottery.

**Comment by progserega** [comment]
I am exit from beta-channel on google-play, remove 1.5.6, install 1.5.4 and element start successfully. 
So, I think before fix this major bug - we can not release 1.5.6 to main channel update from beta-channel.

**Comment by progserega** [comment]
duplicate #7516
